### ISSUE TITLE
"Next session in" displaying 0

### ISSUE THREAD

**Issue by SpeeterYT** [slowlydev/f1-dash#140]
FP3 ended one hour ago, Q1 starts in 1:30h, but the "Next session in" ist just zeros. I also had the same issue one week ago during the hungarian GP. I am on chrome 127 and Firefox 128

**Comment by slowlydev** [comment]
Yes I also noticed that, its probably a combination of caching between nextjs and the api-backend.
